utilities/ImpactUtilities.groovy

```diff
@@ -700,6 +700,8 @@ def saveStaticLinkDependencies(String buildFile, String loadPDS, String member,
 
 		// Store logical file and indirect dependencies to the outputs collection
 		metadataStore.getCollection("${props.applicationOutputsCollectionName}").addLogicalFile( logicalFile );
+	} else {
+		if (props.verbose) println "*** Scanning load module for $buildFile was skipped due to prior condition (no metadatastore connection, build in error)."
```

Comment:
Is this "no metadatastore connection, build in error" a complete list of errors, or suggestions? Maybe some more info should be added.